From 0081518044fcb2b1db799589d2adffe0a14f798d Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Sun, 4 Nov 2018 13:10:42 -0800
Subject: [PATCH] Remove error.h include from lib/system.h

We are only building libelf, which does not need error.h, and
e893aab0d1 had the side-effect of adding error.h to lib/system.h.
---
 lib/system.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/system.h b/lib/system.h
index 292082bd..92033355 100644
--- a/lib/system.h
+++ b/lib/system.h
@@ -30,7 +30,6 @@
 #define LIB_SYSTEM_H	1
 
 #include <errno.h>
-#include <error.h>
 #include <stddef.h>
 #include <stdint.h>
 #include <sys/param.h>
-- 
2.19.1

